docs/source/releases/next.rst

```diff
@@ -11,7 +11,10 @@ Release documentation: `Avocado 109.0
 Users/Test Writers
 ==================
 
-*
+* The installation of Avocado from distribution's packages, changed
+  when compared to the previous version due to changes in repositories
+  and support for "package modules" in some distributions.  Please
+  refer to the installation guide for updated instructions.
```

Comment:
Since you are referencing to the installation guide here, I would add a `ref:` here to the new installation instructions. IMO, it would be easier for readers to find the changes.